src/gennylib/test/context_test.cpp

```diff
@@ -112,23 +114,30 @@ TEST_CASE("loads configuration okay") {
 
     SECTION("Valid YAML") {
         auto yaml = NodeSource(R"(
-SchemaVersion: 2018-07-01
-Clients:
-  Default:
-    URI: 'mongodb://localhost:27017'
-Actors:
-- Name: HelloWorld
-  Type: Nop
-  Count: 7
-        )",
+            SchemaVersion: 2018-07-01
+            Clients:
+              Default:
+                URI: 'mongodb://localhost:27017'
+            Actors:
+            - Name: HelloWorld
+              Type: Nop
+              Count: 7
+            )",
                                "");
 
         WorkloadContext w{yaml.root(), orchestrator, cast};
         auto& actors = w["Actors"];
     }
 
     SECTION("Invalid Schema Version") {
-        auto yaml = NodeSource("SchemaVersion: 2018-06-27\nClients: {Default: {URI: 'mongodb://localhost:27017'}}\nActors: []\n", "");
+        auto yaml = NodeSource(R"(
```

Comment:
What is an R"" string? It looks like we've included indentation in this string where there wasn't any

Reply:
It's a [raw string literal](https://en.cppreference.com/w/cpp/language/string_literal), which can be used to "avoid escaping of any character". Most other NodeSource initializations in this file use raw string literal instead of escaping the new lines, so I converted this one. The relative indentation levels between lines are preserved.

Reply:
It's a [raw string literal](https://en.cppreference.com/w/cpp/language/string_literal), which can be used to "avoid escaping of any character". Most other NodeSource initializations in this file use raw string literal instead of escaping the new lines, so I converted this one. The relative indentation levels between lines are preserved.